src/ConfigResolver/index.ts

```diff
@@ -0,0 +1,91 @@
+import inclusion from 'inclusion'
+import { pathToFileURL } from 'url'
+import { ReporterContract, Refiner, NamedReporterContract } from '@japa/core'
+import { Config, NormalizedConfig } from '../Contracts'
+
+/**
+ * Wrap the value inside an array if it's not one
+ */
+function arrayify<T>(value: T | T[]): T[] {
```

Comment:
How about using `toArray` for the function name?

Reply:
Ahah okay, definitely looks better